src/NuGet.Clients/NuGet.CommandLine/NuGetCommand.resx

```diff
@@ -351,6 +351,33 @@ nuget push foo.nupkg -Timeout 360</value>
   <data name="PushCommandUsageSummary" xml:space="preserve">
     <value>&lt;package path&gt; [API key] [options]</value>
   </data>
+  <data name="SearchCommandDescription" xml:space="preserve">
+    <value>Searches a given source using the query string provided. If no sources are specified, all sources defined in %AppData%\NuGet\NuGet.config are used. If NuGet.config specifies no sources, uses the default NuGet feed.</value>
```

Comment:
Is the thing about the default NuGet feed actually correct? :)

Reply:
I'd lifted this from `nuget list`, since I'd also lifted the `GetEndpointsAsync` method from there and assumed that the behavior (and thus description) for this would be the same. I'll remove that part of the description in case it isn't true, since I'm not actually aware of how that is handled.